man/systemctl.xml

```diff
@@ -2209,6 +2209,11 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
 
           <para>Option <option>--force</option> provides another way to override inhibitors.</para>
 
+          <para>If <varname>EnforceInhibitors=yes</varname> is set in
+          <citerefentry><refentrytitle>logind.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+          then inhibitor locks will be enforced regardless of the privilege leves of the requestor. Option
+          <option>--force --force</option> provides a way to override inhibitor locks in this case.</para>
```

Comment:
```suggestion
          <para>If <varname>EnforceInhibitors=yes</varname> is set in
          <citerefentry><refentrytitle>logind.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
          and <option>--force</option> is not used, inhibitor locks will be enforced regardless of the privilege levels of the requestor.</para>
```

Reply:
I think it's better to recommend this combination, which is more reasonable than `--force --force`